editor/editor_help.cpp

```diff
@@ -1782,7 +1782,7 @@ void EditorHelpBit::_bind_methods() {
 void EditorHelpBit::_notification(int p_what) {
 
 	if (p_what == EditorSettings::NOTIFICATION_EDITOR_SETTINGS_CHANGED) {
-		add_style_override("panel", EditorNode::get_singleton()->get_gui_base()->get_stylebox("ScriptPanel", "EditorStyles"));
+		add_style_override("panel", get_stylebox("ScriptPanel", "EditorStyles"));
```

Comment:
Line 1802 is inside the initiator. since it is not part of the editor_node tree it won't load the editor_theme style box there. It will just load the default theme.